kite-web-element/src/router.js

```diff
@@ -23,6 +25,8 @@ const router = new Router({
             next({
               path: '/containers'
             });
+          } else {
```

Comment:
What does the empty `next()` do?

Reply:
An empty next() moves the router to the next hook in the route pipeline. In english, a next() call basically tells the browser they can continue on to whatever page they requested.

Here is the relevant doc, see the bottom for entire route pipeline - https://router.vuejs.org/guide/advanced/navigation-guards.html.